dir.props

```diff
@@ -90,6 +87,9 @@
     <CoreFxExpectedPrerelease>rc3-24015-00</CoreFxExpectedPrerelease>
   </PropertyGroup>
 
+  <!-- Import packaging props -->
+  <Import Project="$(MSBuildThisFileDirectory)Packaging.props"/>
+
```

Comment:
Yes since it will consume CoreFxExpectedPrerelease.  I can make that change in CoreFx as well.